content/developer/reference/cli.rst

```diff
@@ -639,6 +653,8 @@ Here is a sample file:
     https://www.postgresql.org/docs/9.2/static/libpq-connect.html#AEN38208
 .. _Werkzeug's proxy support:
     https://werkzeug.palletsprojects.com/en/0.16.x/middleware/proxy_fix/#module-werkzeug.middleware.proxy_fix
+.. _set_real_ip_from:
+    https://nginx.org/en/docs/http/ngx_http_realip_module.html
```

Comment:
No need for a hyperlink alias here. It's fine as it is, but the link could be directly in the paragraph to ease maintenance.